### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/blog-source#13] [General]
# about/

杰哥的{运维，编程，调板子}小笔记

https://jia.je/about/

**Comment by 996refuse** [comment]
路过

**Comment by nan0in** [comment]
师傅的博客有rss吗？

**Reply by jiegec** [reply]
https://jia.je/feed.xml

**Comment by wokdevn** [comment]
博主现在的研究方向是什么呢？从SPEC26分析看到这个博客，感觉分析内容挺多的，后续针对SPEC的分析还会继续做吗。

**Reply by jiegec** [reply]
> 博主现在的研究方向是什么呢？从SPEC26分析看到这个博客，感觉分析内容挺多的，后续针对SPEC的分析还会继续做吗。

SPEC CPU 2026 Rate 是分析完了，后续有空可能会把 Speed 补上，还有哪些感兴趣但还没分析的，欢迎提供灵感。
